README.md

```
@@ -1,96 +1,349 @@
+<h2 align="center">
+ <br>
+ <img src="docs/thumbnail.png" alt="Groq Code CLI" width="400">
+ <br>
+ <br>
+ Groq Code CLI: A highly customizable, lightweight, and open-source coding CLI powered by Groq for instant iteration.
+ <br>
+</h2>
 
-A TypeScript-based CLI tool powered by Groq's lightning-fast inference for interactive coding assistance. Built with React Ink for a modern terminal UI experience.
+<p align="center">
+ <a href="https://github.com/build-with-groq/groq-code-cli/stargazers"><img src="https://img.shields.io/github/stars/build-with-groq/groq-code-cli"></a>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)